src/commander.ts

```diff
@@ -18,8 +18,8 @@ export interface RunOptions {
     params: string[];
     startBlock: string;
     stopBlock: string;
-    substreamsApiToken: string;
-    authIssueUrl: string;
```

Comment:
Why remove the `authIssueUrl`?

Reply:
Ok I understand this was only for jwt

Reply:
Yep, it's not required anymore since there's no auth